ckanext/switzerland/logic.py

```diff
@@ -190,7 +189,7 @@ def ogdch_clean_harvester_jobs(context, data_dict):
         all_sources = model.Session.query(HarvestSource.id).all()
         sources_to_clear = list(itertools.chain(*all_sources))
 
-    log.info('Got sources to clear: {}'.format(','.join(sources_to_clear)))
+    log.info('Harvest sources clearing called for: {}'.format(','.join(sources_to_clear)))
```

Comment:
Please change the wording to the specific action that is done here.

Reply:
Done